app/controllers/catalog_controller.rb

```diff
@@ -167,6 +167,58 @@ class CatalogController < ApplicationController
     config.add_show_field 'url_suppl_display_ssm', label: 'More Information'
     config.add_show_field 'lc_callnum_display_ssm', label: 'Call number'
     config.add_show_field 'id', label: 'Catkey'
+    config.add_show_field 'general_note_ssm', label: 'Note'
```

Comment:
@cdmo it is a minor thing but would you mind changing the order of the fields depending on here: https://github.com/psu-libraries/psulib_blacklight/wiki/Blacklight-Catalog-Fields#single-item-view-labels?